### PR TITLE
Fix/sdl 0189 fixes for last comments

### PR DESCRIPTION
**Pull request by LiliiaShlikhtLuxoft**
Changes according comment:
- minor changes
- revert HScrollMenu file's changes
- conditions with OUT_OF_BOUND_RESET_PERIOD
- returned this.props.sliderData in Slider file

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -125,7 +126,9 @@ class HMIApp extends React.Component {
     }
 
     changeResetPeriod(event) {
-        store.dispatch(resetTimeout(event.target.value));   
+        if (event.target.value > OUT_OF_BOUND_RESET_PERIOD) {
```
Comment (ValeriiMalkov) [inline src/index.js:129]: @LiliiaShlikhtLuxoft @YuriiYefimenko Seems check is incorrect? Should be `event.target.value > OUT_OF_BOUND_RESET_PERIOD`. In other case input value should be replaced with default. And check `OUT_OF_BOUND_RESET_PERIOD` should be erased from other place

File: src/css/components/_h-scroll-menu.scss [hunk 2]
```diff
@@ -43,5 +43,4 @@ $calc-width-1: (($box-width-1) - (($margin-size-1) / 1));
 }
 .hscrollmenu .hscrollmenu-block {
     @include h-scroll-item-per-line();    
-}
-
+}
```
Comment (ValeriiMalkov) [inline src/css/components/_h-scroll-menu.scss:46]:
```suggestion
}

```

**Review by ValeriiMalkov — CHANGES_REQUESTED** [review]

**Review by YuriiYefimenko — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (LiliiaShlikhtLuxoft) [reply src/index.js]: @ValeriiMalkov fixed

File: src/css/components/_h-scroll-menu.scss (hunk 2 above)
Comment (LiliiaShlikhtLuxoft) [reply src/css/components/_h-scroll-menu.scss]: @ValeriiMalkov fixed

File: src/js/Alert.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import {AlertButtons} from './containers/Buttons';
 import Image from './Templates/Shared/Image'
 import StaticIcon from './Templates/Shared/StaticIcon'
 
+
```
Comment (ValeriiMalkov) [inline src/js/Alert.js:8]: @LiliiaShlikhtLuxoft Remove empty line

File: src/js/Alert.js [hunk 4]
```diff
@@ -29,4 +30,4 @@ export default class Alert extends React.Component {
             </div>
         )
     }
-}
+}
```
Comment (ValeriiMalkov) [inline src/js/Alert.js:33]:
```suggestion
}

```

**Review by ValeriiMalkov — CHANGES_REQUESTED** [review]

File: src/js/Alert.js (hunk 3 above)
Comment (LiliiaShlikhtLuxoft) [reply src/js/Alert.js]: @ValeriiMalkov fixed

File: src/js/Alert.js (hunk 4 above)
Comment (LiliiaShlikhtLuxoft) [reply src/js/Alert.js]: @ValeriiMalkov fixed

**Review by ValeriiMalkov — APPROVED** [review]

**Pull request merged**